playground/components/header.tsx

```diff
@@ -84,6 +85,33 @@ export const Header: Component<{
               hidden: !showMenu(),
             }}
           >
+            <button
+              type="button"
+              onClick={props.formatCode}
+              class="md:text-white flex flex-row space-x-2 items-center md:px-3 px-2 py-2 focus:outline-none focus:ring-1 rounded opacity-80 hover:opacity-100"
+              classList={{
+                'rounded-none	active:bg-gray-300 hover:bg-gray-300 dark:hover:text-black focus:outline-none focus:highlight-none active:highlight-none focus:ring-0 active:outline-none':
+                  showMenu(),
+              }}
+              title="Format current document"
+            >
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                class="h-6 w-6"
+                fill="none"
+                viewBox="0 0 24 24"
+                stroke="currentColor"
+              >
+                <path
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                  stroke-width="2"
+                  d="M4 6h16M4 12h16M4 18h7"
+                />
+              </svg>
```

Comment:
It looks like this icon is part of `heroicons`, if yes, then I'd suggest using [`solid-heroicons`](https://github.com/amoutonbrady/solid-heroicons) which I believe is already in the project.

```ts
import { Icon } from 'solid-heroicons';
import { menuAlt2} from 'solid-heroicons/outline'; // or from 'solid-heroicons/solid'
```

```suggestion
              <Icon  path={menuAlt2}  class="h-6 w-6"  />
```

Looking at [Stackblitz](https://stackblitz.com/), another option would be to directly set it to the prettier icon:
![image](https://user-images.githubusercontent.com/17355226/153410185-c25185f4-5200-4dda-ad66-4c50258d2fae.png)

What do you think?

Reply:
I also thought about using the Prettier icon instead, but wasn't sure where I can get it.
I'll try it though!

Reply:
At least the icon speaks more of the intent than the previous one which was the left alignment icon 😅 
Thanks!